code/game/objects/items/his_grace.dm

```diff
@@ -12,7 +12,7 @@
 	lefthand_file = 'icons/mob/inhands/equipment/toolbox_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/toolbox_righthand.dmi'
 	icon = 'icons/obj/items_and_weapons.dmi'
-	w_class = WEIGHT_CLASS_GIGANTIC
+	w_class = WEIGHT_CLASS_NORMAL
 	force = 12
 	total_mass = TOTAL_MASS_NORMAL_ITEM // average toolbox
```

Comment:
Remove this line for me thanks, I mistakengly gave it the average normal sized item whack stamina cost (the default for each item is w_class * 1.25, but is overridden by this var) instead of that of a bulky object, and this should be redunant now that its w_class has been changed to match that of a toolbox.

Reply:
Done